tests/Select.test.tsx

```diff
@@ -1458,6 +1458,15 @@ describe('Select.Basic', () => {
     expect(wrapper.find('.rc-select-selection-item')).toHaveLength(0);
   });
 
+  it('reset value to null should reset display value', () => {
+    const wrapper = mount(<Select value="light" />);
+    expect(wrapper.find('.rc-select-selection-item').text()).toEqual('light');
+
+    wrapper.setProps({ value: null });
```

Comment:
https://github.com/ant-design/ant-design/issues/20979#issuecomment-575896823
`undefined` is uncontrolled, but null is not.

Reply:
emmm, 
agree with you

Reply:
bug set to `null` can clear the select in 3.x

Reply:
I'm not sure about this in 3.x, I need to check the code the day after tomorrow to understand it. Maybe someone else knew this before me.

Reply:
https://codesandbox.io/s/antd-select-clear-0yurp

emmm
seems in 3.x, set to `null` is not a **clear** action